js/components/App.js

```diff
@@ -24,6 +38,8 @@ export default Relay.createContainer(App, {
             node {
               id,
               name,
+              enabled,
+              ${ToggleWidgetEnableMutation.getFragment('widget')},
```

Comment:
This is perhaps not relevant to this PR, but why is including this necessary? Aren't the properties from the fragment already here?